pkg/controller/querier/querier.go

```diff
@@ -34,14 +35,16 @@ type ControllerQuerier interface {
 
 type controllerQuerier struct {
 	networkPolicyInfoQuerier querier.ControllerNetworkPolicyInfoQuerier
+	endpointQuerier          networkpolicy.EndpointQuerier
```

Comment:
I think I left a comment about this before. could you point me to the code that relies on `controllerQuerier` embedding a `networkpolicy.EndpointQuerier` member? because I couldn't find it and it seems to me all the changes to this file are actually unnecessary (I haven't tested it so I could be wrong)

Reply:
Actually you are correct, it is never used